script.js

```diff
@@ -61,130 +63,109 @@ app = {
   }
 }
 
+var margin = {
+    left: 75,
+    right: 50,
+    top: 50,
+    bottom: 75
+};
+
 function Chart(selector) {
-  var chart = this;
 
-  // SVG and MARGINS
+  var chart = this;
 
   var margin = {
-    top: 15, right: 15, bottom: 40, left: 45
+    left: 75,
+    right: 50,
+    top: 50,
+    bottom: 75
   };
 
-  chart.width = 600 - margin.left - margin.right;
-  chart.height = 400 - margin.top - margin.bottom;
 
-  chart.svg = d3.select(selector)
+chart.width = 800 - margin.left - margin.right;
+chart.height = 500 - margin.top - margin.bottom;
+
+chart.svg = d3.select(selector)
     .append('svg')
     .attr('width', chart.width + margin.left + margin.right)
     .attr('height', chart.height + margin.top + margin.bottom)
     .append('g')
     .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
+    
+//Scales - Hopefully 
 
-  // SCALES
+// SCALES
 
   chart.x = d3.scaleLinear()
-    .domain([0, d3.max(app.data, function (d) { return d.total_fertility; })])
+    .domain([d3.min(app.data, function (d) { return d.lon; }),d3.max(app.data, function (d) { return d.lon; })])
     .range([0, chart.width])
     .nice();
 
   chart.y = d3.scaleLinear()
-    .domain([0, d3.max(app.data, function (d) { return d.life_expectancy; })])
+    .domain([d3.min(app.data, function (d) { return d.lat; }),d3.max(app.data, function (d) { return d.lat; })])
     .range([chart.height, 0])
     .nice();
 
-  chart.r = d3.scaleSqrt()
-    .domain([0, d3.max(app.data, function (d) { return d.population; })])
-    .range([0, MAX_RADIUS]);
-
-  chart.color = d3.scaleOrdinal(d3.schemeCategory10);
-
-  // AXES
-
-  var xAxis = d3.axisBottom()
-    .scale(chart.x);
-
-  var yAxis = d3.axisLeft()
-    .scale(chart.y);
-
-  chart.svg.append('g')
-    .attr('class', 'x axis')
-    .attr('transform', 'translate(0,' + chart.height + ')')
-    .call(xAxis)
-    .append('text')
-    .attr('y', 30)
-    .attr('x', chart.width)
-    .style('text-anchor', 'end')
-    .style('fill', '#000')
-    .style('font-weight', 'bold')
-    .text('Fertility (births per woman)');
-
-  chart.svg.append('g')
-    .attr('class', 'y axis')
-    .call(yAxis)
-    .append('text')
-    .attr('transform', 'rotate(-90)')
-    .attr('dy', '.71em')
-    .attr('y', -35)
-    .attr('x', 0)
-    .style('text-anchor', 'end')
-    .style('fill', '#000')
-    .style('font-weight', 'bold')
-    .text('Life expectancy (years)');
-
-  // YEAR LABEL
-
-  chart.svg.append('text')
-    .attr('class', 'year')
-    .attr('x', chart.width / 2)
-    .attr('y', chart.height / 2)
-    .attr('dy', '.35em')
-    .style('text-anchor', 'middle')
-    .style('font-size', '230px')
-    .style('font-weight', 'bold')
-    .style('opacity', 0.2)
-    .text(app.options.year);
+  chart.sidelength = d3.scaleSqrt()
+    .domain([d3.min(app.data, function (d) { return d.pop; }), d3.max(app.data, function (d) { return d.pop; })])
+    .range([25, 60]);
```

Comment:
If the range starts at 25, that throws off the scale. Basically that's giving each state those first 25 pixels for free, thus why Wyoming is so big. Suggest that both the domain and range should start at 0 in order to accurately size the squares.